packages/sdk/src/models/RestaurantTagEnum.ts

```diff
@@ -19,7 +19,7 @@
  */
 export enum RestaurantTagEnum {
   FastFood = 'fast food',
-  Wegaska = 'wegańska',
```

Comment:
Lepiej te pliki SDK zostawić tak jak są i w razie potrzeby edytować serwer, bo przy kolejnym `yarn gen:sdk` to co zmieniłaś i tak się wyzeruje.